.gitignore

```diff
@@ -17,3 +17,7 @@ npm-debug.log
 yarn-debug.log
 yarn-error.log
 dist/
+
+# Ignore actionlint binary and any leftover directories related to installing it
+actionlint*
```

Comment:
Didn't realize this when I made the suggestion, but this will cause `actionlint.sh` to be ignored too. It will still work because the file is already tracked, but it could be confusing. You could:
1. Change this to `actionlint` and remove the line below - could cause somebody to accidentally commit the tarball or tmp dir
2. Replace this with the tarball and tmp dir prefix (and remove the line below)
3. Do nothing

NAB 😊

Reply:
good catch, fixed.